src/org/jitsi/impl/neomedia/MediaStreamStatsImpl.java

```diff
@@ -1641,4 +1641,24 @@ private MediaStreamStats2Impl getExtended()
     {
         return mediaStreamImpl.getMediaStreamStats();
     }
+
+    /**
+     * Notifies listeners that a transport-wide-cc packet was received.
+     * Listeners may include Remote Bitrate Estimators or Bandwidth Estimators
+     * @param tccPacket
+     */
+    public void tccReceived(RTCPTCCPacket tccPacket)
```

Comment:
Can you please add an @Override annotation (since this implements an interface method)